components/builder-graph/src/error.rs

```diff
@@ -33,12 +32,10 @@ pub enum Error {
     DbTransaction(postgres::error::Error),
     HabitatCore(hab_core::Error),
     IO(io::Error),
-    PackageInsert(postgres::error::Error),
```

Comment:
I love to see unused errors getting cleaned up. I really wish the compiler found this for us!